### PR TITLE
use torus provider as default

### PR DESCRIPTION
**Pull request by andrewdamelio**
- update torus to v1
- if window.web3 is not available, use torus as the web provider

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 🍪
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/zr5hj14**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

File: packages/augur-ui/src/modules/auth/actions/login-with-fortmatic.ts [hunk 1]
```diff
@@ -10,13 +10,13 @@ import { getNetworkId } from 'modules/contracts/actions/contractCalls';
 import { windowRef } from 'utils/window-ref';
 import { AppState } from 'store';
 
-const getFormaticNetwork = (networkId: string): false | string   => {
+const getFormaticNetwork = (networkId: string): string   => {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/auth/actions/login-with-fortmatic.ts]: Can this be moved out to a helper to convert networkId to network name

File: packages/augur-ui/src/modules/app/actions/init-augur.ts [hunk 2]
```diff
@@ -214,10 +217,39 @@ export function connectAugur(
       preloadAccount(ACCOUNT_TYPES.TORUS);
     }
 
-    let provider;
-    if (windowApp.web3) {
-      provider = new Web3Provider(windowApp.web3.currentProvider);
+    let provider = null;
+    let networkId = null;
+
+    if (process.env.ETHEREUM_NETWORK === 'kovan') {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/actions/init-augur.ts]: Can this be moved out to helper. Maybe with helper to convert networkId to network name

File: packages/augur-ui/src/modules/auth/actions/login-with-fortmatic.ts (hunk 1 above)
Comment (pgebheim) [reply packages/augur-ui/src/modules/auth/actions/login-with-fortmatic.ts]: There's already a mapping for this, yah

**Review by pgebheim — COMMENTED** [review]
Currently this code is doing the Kovan / Mainnet checks inline -- but I want to point out that the name here for ETHEREUM_NETWORK isn't really a network, its the name of a config.

We are likely going to make a `demo` config that is running against Kovan. We are also likely to test against Rinkeby etc, so we shouldn't hardcode this to kovan, at minimum. I think we should put the network ID on the SDKConfiguration (in network.json / env) for anything except dev nets and then this can just be pulled off the config.

See: `isDevNetworkId` from augur-artifacts/index.ts

**Review by bthaile — CHANGES_REQUESTED** [review]
consolidate logic for translating network/networkIds

**Review by bthaile — APPROVED** [review]
lgtm

**Review by pgebheim — APPROVED** [review]
